numpy/random/generator.pyx

```diff
@@ -3361,6 +3361,19 @@ cdef class Generator:
         tol : float, optional
             Tolerance when checking the singular values in covariance matrix.
             cov is cast to double before the check.
+        method : { 'svd', 'eigh', 'cholesky'}, optional
+            The cov input is used to compute a factor matrix A such that
+            transpose(A) * A = cov. This argument is used to select the method
+            used to compute the factor matrix A. The default method 'svd' is
+            the slowest, while 'cholesky' is the fastest but less robust than
+            the slowest method. The method `eigh` uses eigen decomposition to
+            compute A and is faster than svd but slower than cholesky.
+        use_factor : bool, optional
+            If set to True then cov argument is treated as a precomputed factor
+            matrix A such that np.dot(A * transpose(A)) == cov is True, with the
```

Comment:
`np.dot(A * transpose(A)) == cov` This cannot be right.  Should match whatever is used above.

Reply:
you're right, my mistake. Thanks for pointing that out.